intdataplane/int_dataplane.go

```diff
@@ -427,6 +439,15 @@ func (d *InternalDataplane) loopUpdatingDataplane() {
 				mgr.OnUpdate(ifaceAddrsUpdate)
 			}
 			d.dataplaneNeedsSync = true
+		case ifaceRoutesUpdate := <-d.ifaceRouteUpdates:
+			log.WithField("msg", ifaceRoutesUpdate).Info("Received interface routes update")
+			for _, mgr := range d.allManagers {
```

Comment:
I think you can remove this loop, there aren't any managers listening for route updates, and, since we use a type switch in the `OnUpdate()` methods to unpack the input, sending in a `string` is ambiguous.